Simplenote/Classes/SPNoteListViewController.m

```diff
@@ -50,6 +42,13 @@ @interface SPNoteListViewController () <ABXPromptViewDelegate,
                                         UITextFieldDelegate,
                                         SPTransitionControllerDelegate>
 
+@property (nonatomic, strong) UIBarButtonItem           *addButton;
```

Comment:
I don't like this alignment... :)

Reply:
as an outsider coming in, what don't you like about it?

Reply:
LOL

This kind of "vertical alignment" has been... the source of conversations thru the years. Looks super cool, but has a maintainability fee (that's all, pretty much!).

😄

Reply:
Sorry about this comment @dmsnell. This is like an ongoing conversation that I have with @jleandroperez about his preference of aligning vertically variable names.

I don't like it because if you this, every time you change a variable name or type you need to realign the variables introducing extra noises on diffs and unnecessary changes.

Reply:
are there any standard deterministic auto-formatters for Objective-C/Swift?

have folks discussed using something like that to eliminate the conversations in code review about style/indentation?

Reply:
I've seen tools that auto format code... but... implementing / integrating those, in this particular case, takes effort we can't invest there.

Mostly because we *probably should* invest that time in migrating code over to Swift, and adding unit test!